app/helpers/application_helper.rb

```diff
@@ -1,10 +1,10 @@
 module ApplicationHelper
-  def full_title(page_title = '')                     
+  def full_title page_title = ''
     base_title = "Library Managerment"
-    if page_title.empty?              
-      base_title                      
+    if page_title.empty?
```

Comment:
using ternary operator for writing code briefly

Reply:
fixed